repo/repo-sql-impl/src/main/java/com/evolveum/midpoint/repo/sql/data/common/container/RCaseWorkItem.java

```diff
@@ -66,6 +66,7 @@
     private Set<RCaseWorkItemReference> assigneeRef = new HashSet<>();
     private REmbeddedReference performerRef;
     private String outcome;
+    private String comment;
```

Comment:
Do we really need to search by comment? Maybe full text search would be a better solution.